tests/test_form.py

```diff
@@ -467,6 +467,25 @@ def test_issue158():
     browser.close()
 
 
+def test_issue264():
```

Comment:
I would prefer more meaningful test name (so that one can immediately know which test is about what when skimming through a list of tests, without having to open the corresponding issue each time) and a link to the issue in the docstring, but i'm OK with this if you prefer.

Reply:
Renamed to `test_duplicate_submit_buttons`